package.json

```diff
@@ -111,7 +115,7 @@
     "test": "cypress run",
     "lint": "eslint src",
     "preview": "yarn run build && netlify deploy --dir public",
-    "test:unit": "mocha --require @babel/register src/**/*.test.js",
+    "test:unit": "jest",
```

Comment:
I switched all unit tests to jest instead of mocha because I needed to use full DOM rendering (rather than shallow rendering) in the tests of the `<Link />` component. Jest has better integrations with JSDOM for doing this. There wasn't many unit tests in the site so it didn't take long.